extension/src/lib/zoom.js

```diff
@@ -0,0 +1,50 @@
+/*globals fluid */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+fluid.defaults("gpii.chrome.zoom", {
+    gradeNames: "fluid.modelComponent",
+    model: {
+        magnifierEnabled: false,
+        magnification: 1
+    },
+    invokers: {
+        set: {
+            funcName: "gpii.chrome.zoom.set",
+            args: "{arguments}.0"
+        }
+    },
+    modelListeners: {
+        "*": {
```

Comment:
Also, if you use "" you can get rid of the "{that}" argument and just accept the default which will then just consist of the whole model - also change above. But in this case it looks like you only really want to listen to changes to "magnifierEnabled"